lib/vernier/output/firefox.rb

```diff
@@ -385,6 +385,20 @@ def string_table
 
         private
 
+        def pretty_name(name)
+          if name.empty?
+            tr = ObjectSpace._id2ref(@ruby_thread_id)
```

Comment:
if we somehow didn't get the inspect value in the native extension, fallback to doing it in ruby.

Reply:
TO DO - ensure that we rescue the exception.